setup.cfg

```diff
@@ -27,4 +27,5 @@ omit =
 	syft/frameworks/torch/nn/rnn.py
 	syft/workers/websocket_server.py
 	test/efficiency/*
+	test/notebooks/*
```

Comment:
Since the notebook tests no longer run in the same action as the rest of the tests, they also need to be excluded from the test coverage checks.